packages/forma-36-website/src/pages/guidelines/copy/grammar/index.mdx

```diff
@@ -0,0 +1,3 @@
+# Grammar and mechanics
```

Comment:
I'd go with just `grammar`, or, make all of the directory names and urls match this title